spec/controllers/api/v1/book_suggestion_controller_spec.rb

```diff
@@ -0,0 +1,46 @@
+describe Api::V1::BookSuggestionController, type: :controller do
+  include_context 'Authenticated User'
+  include Devise::Test::ControllerHelpers
+
+  describe 'POST #create' do
+    context 'When creating a BookSuggestion with a valid user' do
+      let!(:book_sugg) { create(:book_suggestion, user: user) }
+      let(:post_create) do
```

Comment:
This should be the `subject` of your `describe 'POST #create'`. And try to define it out of `context`